From 6fe702170dcaae04c9867de47615b5b6d2547681 Mon Sep 17 00:00:00 2001
From: "faerietree (ciryat)" <radagast@ciry.at>
Date: Fri, 27 Apr 2018 06:38:01 +0000
Subject: [PATCH] Use utf8 instead of utf8mb4 to fix MariaDB, MySQL <=5.7.6
 error "max key length is 767 bytes".

This means storing data like Emoji smileys is not possible.
---
 config/database.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/database.php b/config/database.php
index cab5d06..5ed066d 100644
--- a/config/database.php
+++ b/config/database.php
@@ -47,8 +47,8 @@ return [
             'username' => env('DB_USERNAME', 'forge'),
             'password' => env('DB_PASSWORD', ''),
             'unix_socket' => env('DB_SOCKET', ''),
-            'charset' => 'utf8mb4',
-            'collation' => 'utf8mb4_unicode_ci',
+            'charset' => 'utf8',
+            'collation' => 'utf8_unicode_ci',
             'prefix' => '',
             'strict' => true,
             'engine' => null,
-- 
2.16.2

